tests/test_snippets.py

```diff
@@ -78,7 +80,7 @@ def run_via_rustpython(filename, test_type):
     env['RUST_LOG'] = '{},cargo=error,jobserver=error'.format(log_level)
     env['RUST_BACKTRACE'] = '1'
     subprocess.check_call(
-        ['cargo', 'run', '--release', filename], env=env)
+        ['cargo', 'run', '--release' if RELEASE else '', filename], env=env)
```

Comment:
Our Travis CI needs to run this in release mode. It would be great if you can also fix the `.travis.yml` file and let it run in release. But I know Travis is not a requirement for this task. So if you choose to leave Travis config for later, please make this run in release mode by default.